src/jarabe/desktop/schoolserver.py

```diff
@@ -56,7 +61,25 @@ def _generate_serial_number():
     return serial
 
 
-def _store_identifiers(serial_number, uuid_, backup_url):
+def _get_history_for_serial(serial_number):
+    identifier_path = os.path.join(env.get_profile_path(), 'identifiers')
+    if not os.path.exists(identifier_path):
+        os.mkdir(identifier_path)
```

Comment:
These three lines are duplicate of code in `_store_identifiers`, please refactor.